src/SmartComponents/BaselinesTable/BaselinesTable.js

```diff
@@ -55,13 +60,17 @@ class BaselinesTable extends Component {
     }
 
     renderKebab() {
-        const { baselineTableData } = this.props;
+        const { baselineTableData, addBaselineUUID } = this.props;
         let tableWithKebab = [];
 
         baselineTableData.forEach(function(baseline) {
             let row = [];
-            baseline.forEach(function(data) {
-                row.push(data);
+            baseline.forEach(function(data, index) {
+                if (index === 0) {
+                    row.push(<div><a onClick={ addBaselineUUID.bind(this, baseline.id) }>{ data }</a></div>);
```

Comment:
@johnsonm325 Wondered if you had any ideas here; this onClick won't render in the table. I assume it is due to some processing being done by the component that consumes these rows.